az-digital/az_quickstart
# Allow id, name classes to headings, plus additional button classes in ckeditor (Closes #762)

This PR makes "name" and "id" allowable for headings, divs, and spans, and also expands the allowable button classes

## Description
This PR changes the allowed html for the standard text format modules/custom/az_media/config/install/filter.format.az_standard.yml

Specifically adding `name=""` and `id=""` to heading, div, and span tags. This will allow content editors to create anchor links pointing to ids on these tags.

I also went ahead and added the additional button stylings that @barrettbaffert mentions on [this ticket](https://github.com/az-digital/arizona-gatsby/issues/35) for the Admissions project. This allows more advanced editors to add button classes in the source for:
`btn-block btn-lg btn-sm btn-arrow`

![image](https://user-images.githubusercontent.com/10873398/119239232-950b4200-bafc-11eb-821f-ac32914d2b16.png)


## Related Issue
#762 
Also see: https://github.com/az-digital/arizona-gatsby/issues/35

## How Has This Been Tested?
Tested locally and in Probo

To test:

1. Create a page
2. Add a paragraph using the standard text editor
3. Add headings and text (can also test spans and divs)
3. View Source
4. Add name and id to heading (or span/div) tags
5. Save page and then edit again to verify that name and id not stripped from source code
6. Create an anchor link pointing to one or more of your ids
7. Verify anchor link works
8. Add buttons and apply button classes in the source code (btn-lg btn-sm btn-block btn-arrow)
9. Verify buttons appear as expected after saving

## Types of changes


## Checklist:
